src/core/symbology/qgsmarkersymbollayer.cpp

```diff
@@ -2487,29 +2494,37 @@ QRectF QgsSvgMarkerSymbolLayer::bounds( QPointF point, QgsSymbolRenderContext &c
   }
   strokeWidth = context.renderContext().convertToPainterUnits( strokeWidth, mStrokeWidthUnit, mStrokeWidthMapUnitScale );
 
-  //need to get colors to take advantage of cached SVGs
```

Comment:
can you leave this comment? I found it useful to know why the colors are being decoded here

Reply:
@nyalldawson , will re-add -- thanks for the review, the SVG marker code logic can be a bit of a PITA.